pegasus/routes/hoc_routes.rb

```diff
@@ -45,6 +45,13 @@
   launch_tutorial(tutorial, company: company)
 end
 
+# Specially track links from our /learn page
+get '/api/hour/begin_learn/:code' do |code|
+  only_for ['code.org', 'csedweek.org', partner_sites].flatten
+  pass unless tutorial = DB[:tutorials].where(code: code).first
+  launch_tutorial_learn(tutorial)
+end
```

Comment:
This doesn't have some of the nonsense in `/api/hour/begin/:code`. Presumably, this nonsense is not necessary?